### PR TITLE
README Update Oct. 2021

### PR DESCRIPTION
**Pull request by RobertScalfani**
Updated the SetupGuideForMacOS.md instructions based on first time setup experience.

- Updated a few out-of-date instructions.
- Updated required Java version from 8 -> 11.
- Added a few troubleshooting options based on issues I had.
- Added the CMS-DRLS VSAC author group membership requirement.

### CODE REVIEW

File: SetupGuideForMacOS.md [hunk 1]
```diff
@@ -156,32 +156,25 @@ Additionally, you must have credentials (api key) access to the **[Value Set Aut
...
-    {
-        contentBase: path.resolve(__dirname, "public"),
-        port: 3005,
-        https: false,
-        host: "0.0.0.0",
-        public: "0.0.0.0",
-    
-        ...
-    ```
+***None***
```
[the remark this answers is not in the document]
Comment (KeeyanGhoreshi) [reply SetupGuideForMacOS.md:172]: I commented this here, but the register step should be added under "start DTR", not config.

File: SetupGuideForMacOS.md [hunk 2]
```diff
@@ -222,15 +215,16 @@ Wait until you see the server is running at 91% and there is no more console out
...
 rm -rf target # not required if you are running test-ehr for the very first time
 rm -rf build # not required if you are running test-ehr for the very first time
-gradle appRun
+gradle bootRun
 gradle loadData
 ```
 Wait until you see the server is running at 91% and there is no more console output before proceeding to [Start keycloak]().
 
-> Optional. Verify test-ehr is running:
+> Optional. Verify test-ehr is running with the following links. You may need to add '/r4' to the end of the links.
```
[the remark this answers is not in the document]
Comment (RobertScalfani) [reply SetupGuideForMacOS.md:223]: Fixed these links, removed references to HAPI UI.

File: SetupGuideForMacOS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (RobertScalfani) [reply SetupGuideForMacOS.md:172]: The register step is detailed in "Register the test-ehr" - I added a reference to that section here.

File: SetupGuideForMacOS.md [hunk 3]
```diff
@@ -55,18 +55,18 @@ Additionally, you must have credentials (api key) access to the **[Value Set Aut
 
 ### Installing core tools on MacOS
 
-#### Install AdoptOpenJDK8
+#### Install AdoptOpenJDK11
 
-> Important note: **Do not skip** these steps, **even if you already have AdoptOpenJDK8 installed on your Mac**, as they will make your life easier when you need to update and/or switch your Mac's default Java version down the line.
+> Important note: **Do not skip** these steps, **even if you already have AdoptOpenJDK11 installed on your Mac**, as they will make your life easier when you need to update and/or switch your Mac's default Java version down the line.
 
 1. Install [sdkman](https://sdkman.io/), a MacOS-native tool for package management.
```
[the remark this answers is not in the document]
Comment (RobertScalfani) [reply SetupGuideForMacOS.md:62]: Updated to allow use of other package managers.

**Review by KeeyanGhoreshi — APPROVED** [review]

**Pull request merged**